package/MDAnalysis/auxiliary/EDR.py

```diff
@@ -362,6 +362,7 @@ def _read_next_step(self) -> EDRStep:
         StopIteration
             When end of auxiliary data set is reached.
         """
+        SINGLE_FRAME = True if self.n_steps == 1 else False
```

Comment:
```suggestion
        SINGLE_FRAME = self.n_steps == 1
```

Reply:
Thanks! I considered doing that, but then decided against it because "explicit is better than implicit". I only have a mild preference here though, happy to change it if you prefer the shorter version

Reply:
What about "Beautiful is better than ugly." or "Simple is better than complex."? ;) Jokes aside, some liners complain about [simplifiable-if-expression](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/simplifiable-if-expression.html).

Reply:
Haha, fair enough - thanks :)